kieserver/image.yaml

```diff
@@ -146,7 +146,7 @@ envs:
     example: "true"
     description: "Setting with true enable the others KIE_KAFKA_EXT environments used by Kafka, this env will be used by system prop org.kie.kafka.server.ext.disable with the opposite value"
   - name: "KIE_SERVER_KAFKA_EXT_TOPICS"
-    example: "event=events"
+    example: "message=topic_name"
     description: "Contains the mapping message/signal=topicName for every topic that needs to be mapped globally, this env will be used by system prop org.kie.server.jbpm-kafka.ext.topics."
```

Comment:
mention here how to provide more than one. do the same for the rhpam template as well.

Reply:
ok